db/post_migrate/20210922064213_alter_bumped_at_indexes_on_topics.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
```

Comment:
Nitpick: can you run `annotate` to update `topic.rb` too? Lots of changes to indexes happen in this file.

Reply:
O right I forgot about it :+1: